01 - Community/People/Josh Plunkett

```diff
@@ -0,0 +1,65 @@
+---
+aliases:
+- Joshua Plunkett
+- Sklore
+- Sklex
+tags: 
+- 
+publish: true
+---
+
+# Josh Plunkett
+
+%% - GitHub: [agathauy](https://github.com/agathauy/) ^github %%
+
+- Discord: `@sklore#8618` ^discord
+
+
+%% - Website: <https://github.com/agathauy> ^website %%
+
+<!-- - [[Publish sites|Publish site]]: ^publish -->
+
+%% Feel free to add a bio below this comment %%
+
+## Author of
+Tutorials for using Obsidian as a TTRPG Campaign Manager. 
+- [Patreon](https://www.patreon.com/JPlunkett)
```

Comment:
@argenos I'd be interested to know what you think about the Patreon link being here instead of at the end ... I think maybe these templates are just more confusing than I had realised :-)

Reply:
I use Patreon as a place to release non-video content so I didn't want to associate it with strictly supporting me fiscally.

Reply:
I see, thanks for clarifying!